docs/04-Reference/03-Database/02-Data-Modelling.md

```diff
@@ -39,7 +39,7 @@ There are several available building blocks to shape your data model.
 * [Relations](#relations) describe interactions between types.
 * Special [directives](#graphql-directives) that cover different use cases are available.
 
-Additionally, a service can contain prepopulated types and fields, referred to as [system artifacts](#system-artifacts). Different [naming conventions](#naming-conventions) define valid names.
+Additionally, a service can contain prepopulated types and fields, referred to as [system artifacts](#system-artifacts-(only-for-legacy-console-projects)). Different [naming conventions](#naming-conventions) define valid names.
```

Comment:
I'm not sure if this sentence should be removed entirely, as it's only relevant to legacy projects

Reply:
It's relevant for all services: #1142.